tracer/src/Datadog.Trace/Agent/Api.cs

```diff
@@ -27,8 +29,12 @@ internal class Api : IApi
         private readonly IDogStatsd _statsd;
         private readonly string _containerId;
         private readonly Uri _tracesEndpoint;
+        private readonly Uri _statsEndpoint;
         private readonly Action<Dictionary<string, float>> _updateSampleRates;
         private readonly bool _isPartialFlushEnabled;
+        private readonly bool _areTracerStatsEnabled;
```

Comment:
Not a blocker, but these field names feel weird to me. Part of it is that I don't like to prefix Boolean values with `is`/`are` if they already end in `Enabled`. Seems redundant.

Even if you do that, this field describes _one_ feature (called "Tracer Stats"), so I would use the singular `is`. Think "is the feature (Tracer Stats) enabled."